rollingshutter.py

```diff
@@ -1,31 +1,46 @@
-def rs(folder,speed=1):
-    
-    from PIL import Image
-    import glob
-    
-    # ENTER YOUR DIRECTORIES AND FILE TYPE HERE
-    frame_dir = "/Users/mattparker/Documents/rollingshuttervideos/" + str(folder) + "/"
-    frame_file = "png"
-    output_dir = "/Users/mattparker/Documents/rollingshuttervideos/"
-    
-    width = 1920
-    height = 1080
+from PIL import Image
+import argparse
+import glob
+import os
+
+
+def rolling_shutter(input_directory, output_directory, extension_type, image_height, image_width, speed):
     
     # Making our blank output frame
-    output_image = Image.new('RGB', (width, height)) 
+    output_image = Image.new('RGB', (image_width, image_height)) 
     
     
     # let us go through the frames one at a time
     
     current_row = 0
-    
-    for filename in glob.glob(frame_dir + "*." + frame_file):
+     
+    for filename in glob.glob(os.path.join(input_directory, "*.", extension_type)):
```

Comment:
`os.path.join(input_directory, "*.", extension_type)` adds a slash between the files and the extension.
Should be `os.path.join(input_directory, "*." + extension_type)`.